table/block_based/full_filter_block_test.cc

```diff
@@ -14,6 +14,8 @@
 #include "util/hash.h"
 #include "util/string_util.h"
 
+#include <set>
```

Comment:
It surprised me as I thought one of the internal linters will report this. Google C++ Style's include order would put STL headers perhaps ahead:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes